test/witan/app/handler_test.clj

```diff
@@ -16,13 +16,15 @@
     (with-redefs [user/user-valid? (fn [username password] true)]
       (let [[status body _] (post* app "/api/login" {:body (json {"username" "support@mastodonc.com" "password" "secret"})})]
         (is (= status 200))
-        (is (contains? body :token)))))
+        (is (contains? body :token))
```

Comment:
If you end up defining schemas for use with compojure-api's `:return` field, then some of these tests could be condensed to a single `(is (s/validate...` - just a thought.